Sources/do-not-disturb/DoNotDisturb.swift

```diff
@@ -12,22 +12,35 @@ public struct DoNotDisturb {
 		DistributedNotificationCenter.default().postNotificationName(NSNotification.Name("com.apple.notificationcenterui.dndprefs_changed"), object: nil, deliverImmediately: true)
 	}
 
-	private static func enable() {
-		set("dndStart", value: 0 as CFPropertyList)
-		set("dndEnd", value: 1440 as CFPropertyList)
-		set("doNotDisturb", value: true as CFPropertyList)
```

Comment:
The reason this was happening is because if the `dndStart` and `dndEnd` are being used, the `doNotDisturb` needs to be set to false, and the system uses the current time to figure out if DND is on or off. So, when setting `dndStart` and `dndEnd` and committing the System changes the `doNotDisturb` back to false.